src/OrchardCore.Themes/TheTheme/Views/ToggleTheme.cshtml

```diff
@@ -1,8 +1,8 @@
-<li class="nav-item nav-item dropdown text-end">
-    <a class="nav-link dropdown-toggle" id="bd-theme" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="@T["Toggle theme"]">
+<li class="nav-item dropdown text-end">
+    <button class="nav-link dropdown-toggle" id="bd-theme" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="@T["Toggle theme"]">
```

Comment:
add `type="button"` Have you tested this out on all screen sizes to make sure it does not break the view?

Reply:
This is used in all other built in themes. I haven't tested it on all screen sizes.

Reply:
I thought this was just a copy paste error.

Reply:
Let do a test please specially on small screen. Let's not assume it'll work since CSS could be training a link not a button for styling.

Reply:
You are right, it did change the styling in mobile view. Added `role` because `type` won't help here.